app/src/display/widgets/Kconfig

```diff
@@ -22,5 +22,11 @@ config ZMK_WIDGET_OUTPUT_STATUS
     default y if BT
     select LVGL_USE_LABEL
     select LVGL_FONT_MONTSERRAT_16
+    
+config ZMK_WIDGET_WPM_STATUS
+    bool "Widget for displaying typed words per minute"
+    depends on !ZMK_SPLIT || ZMK_SPLIT_BLE_ROLE_CENTRAL
+    select LVGL_USE_LABEL
+    select LVGL_FONT_MONTSERRAT_16
```

Comment:
Shouldn't this also set `ZMK_WPM=y` to avoid build errors? I imagine if a user turns on the WPM widget they are going to want `ZMK_WPM` set as well

Reply:
Good Catch, I'll get that added